ethereum/rlp/build.gradle

```diff
@@ -34,7 +34,6 @@ dependencies {
   api project(':util')
 
   implementation 'com.google.guava:guava'
-  implementation 'io.vertx:vertx-core'
```

Comment:
Seems unrelated, but it's a good change.

Reply:
It's related to the library transition, should have been in the prior PR.  This way the EVM lib does not drag in vertx.